src/microbe_stage/components/OrganelleContainer.cs

```diff
@@ -532,6 +538,12 @@ public static void CalculateOrganelleLayoutStatistics(this ref OrganelleContaine
             if (organelleDefinition.HasBindingFeature)
                 container.HasBindingAgent = true;
 
+            if (organelleDefinition.InternalName == "rusticyanin")
```

Comment:
These shouldn't check internal names like this, instead components or another property should be used. Perhaps even an "iron score" field should be added to organelle definition that is dynamically calculated from the specified processes at registry load time.

Reply:
This comment is not addressed.